templates/terraform/custom_flatten/cloudscheduler_job_appenginerouting.go.erb

```diff
@@ -0,0 +1,24 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2019 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// An `appEngineRouting` in API response is useless, so we set config values rather than api response to state.
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData) interface{} {
+	if v == nil {
+		return nil
+	}
+	if v, ok := d.GetOK("app_engine_http_target"); ok && len(v.([]interface{})) > 0 {
```

Comment:
GetOK needs to actually be GetOk

Reply:
So sorry to commit without confirmation! Fixed it, and confirmed the terraform provider compiled by this change works as expected.
f06e040